Dockerfile.nitro

```diff
@@ -0,0 +1,230 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
it'll be good to update the NOTICE to point to where this original Dockerfile was taken from

Reply:
added NOTICE